api/lib/domain/usecases/get-external-authentication-redirection-url.js

```diff
@@ -0,0 +1,26 @@
+module.exports = async function getExternalAuthenticationRedirectionUrl({
+  userAttributes,
+  userRepository,
+  tokenService,
+  settings,
+}) {
+  const { attributeMapping } = settings.saml;
+
+  const externalUser = {
+    firstName: userAttributes[attributeMapping.firstName],
+    lastName: userAttributes[attributeMapping.lastName],
+    samlId: userAttributes[attributeMapping.samlId],
+  };
+
+  const foundUser = await userRepository.getBySamlId(externalUser.samlId);
+
+  if (foundUser) {
+    const token = tokenService.createTokenFromUser(foundUser, 'external');
```

Comment:
cf. le commentaire précédent, 

```suggestion
    const token = tokenService.createAccessTokenFromUser(foundUser, 'external');
```